ci/test.yml

```diff
@@ -71,27 +71,6 @@ core device ui test:
     reports:
       junit: tests/junit.xml
 
-core device test:
-  stage: test
-  <<: *gitlab_caching
-  needs:
-    - core unix frozen debug build
-  variables:
-    TREZOR_PROFILING: 1
```

Comment:
We'll need to enable the coverage data collection in the new `core device test`.